doc/source/whatsnew/v1.2.1.rst

```diff
@@ -10,6 +10,18 @@ including other versions of pandas.
 
 .. ---------------------------------------------------------------------------
 
+Enhancements
+~~~~~~~~~~~~
+
+.. _whatsnew_121.testing.assert_frame_equal:
+
+Added ``check_freq`` argument to ``testing.assert_frame_equal``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
suprised this didn't fail the build, underline has to be equal to the len of the title